### PR TITLE
CMakeScripts updates to support relocatable installs and build options

### PR DESCRIPTION
**Pull request by johnb003**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -371,7 +370,7 @@ CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/cmake-zipconf.h.in ${CMAKE_CURRENT_BI
 
 SET(srcdir ${CMAKE_CURRENT_SOURCE_DIR}/regress)
 SET(abs_srcdir ${CMAKE_CURRENT_SOURCE_DIR}/regress)
-SET(top_builddir ${CMAKE_BINAfindRY_DIR})
+SET(top_builddir ${CMAKE_BINARY_DIR})
```
Comment (johnb003) [inline CMakeLists.txt]: In retrospect, I probably shouldn't have tried to build this patch using github's text editor...

**Comment by johnb003** [comment]
I'm trying with clang c on linux, but I can't reproduce the failures for aes.

**Comment by johnb003** [comment]
I'm gonna need help with the test failures.

**Comment by 0-wiz-0** [comment]
No feedback. We can re-open, or you can make a new one, when you've fixed the Windows build problems this causes (UWP builds are currently broken, but this PR also breaks the non-UWP builds).

**Pull request closed without merging**